README.md

```diff
@@ -301,6 +301,45 @@ import exampleGo from '!!raw-loader!./code-example.go'
 ```
 ````
 
+### Mermaid graphs
+
+Use [mermaid.js](https://mermaid-js.github.io/mermaid/#/) to create graphs and
+diagrams,
+[for example](https://www.ory.sh/docs/kratos/self-service/flows/user-registration).
```

Comment:
When you use "for example" I expect some text that follows to be that example. Here you made "for example" a hyperlink. Please don't do that. 
Also, this is a very dated example that will be gone sooner than later. Please remove that.